BaseStation/VideoStreamer/App.xaml

```diff
@@ -1,9 +0,0 @@
-﻿<Application x:Class="VideoStreamer.App"
```

Comment:
VideoStreamer is no longer a standalone application. It can be used from the BaseStation project.